packages/common/src/TransactionUtils.ts

```diff
@@ -19,6 +21,17 @@ export interface AugmentedSendOptions {
   maxPriorityFeePerGas?: number | string;
 }
 
+interface AugmentedWeb3 extends Web3 {
+  nonces: { [address: string]: number };
+}
+
+export interface executedTransaction {
```

Comment:
nit: types should be cap case
```suggestion
export interface ExecutedTransaction {
```